src/UI/Implementation/Component/Link/Renderer.php

```
@@ -16,16 +14,17 @@ class Renderer extends AbstractComponentRenderer
     /**
      * @inheritdoc
      */
-    public function render(Component\Component $component, RendererInterface $default_renderer)
+    public function render(Component\Component $component, RendererInterface $default_renderer) : string
     {
         $this->checkComponent($component);
 
         if ($component instanceof Component\Link\Standard) {
-            return $this->renderStandard($component, $default_renderer);
+            return $this->renderStandard($component);
         }
         if ($component instanceof Component\Link\Bulky) {
             return $this->renderBulky($component, $default_renderer);
         }
+        return "";
```

Please throw a `LogicException` if we have an unknown component.

---

throw new LogicException("Cannot render: " . get_class($component));